tests/search/native_test.php

```diff
@@ -56,6 +58,30 @@ public function keywords()
 				array(),
 			),
 			array(
+				'baaz*',
+				'all',
+				true,
+				array('\'baaz%\''),
+				array(),
+				array(),
+			),
+			array(
+				'ba*az',
+				'all',
+				true,
+				array('\'ba%az\''),
```

Comment:
This looks weird, but split_keyword has been designed in such a way that it returns these strings in case of partial matches.